post/urls.py

```diff
@@ -0,0 +1,26 @@
+from django.conf.urls import url
+from . import views
+
+app_name = 'post'
+
+urlpatterns = [
+    # /post/
+    #url(r'^$', views.index, name='index'),
```

Comment:
Will clean up. The comments were there more as a reference to myself, just in case. Just a first round. I need to do some refactoring before it becomes final.